lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -501,10 +514,18 @@ def main():
         state = module.params['state']
         control_state(storage_domains_module)
         if state == 'absent':
+            # Pick random available host when host parameter is missing
+            host_param = module.params['host']
+            if not host_param:
+                hosts_list = connection.system_service().hosts_service().list()
```

Comment:
I think it's better to use something like:

```python
host = search_by_attributes(connection.system_service().hosts_service(), status='up')
host_param = host.name if host is not None else None
```

In case there is lot of hosts, it would perform faster.

Reply:
Done